### PR TITLE
Fixing loading bug of old otevents files, Starttime bug (when specific folder name), fix name suggestion for saving

### PR DESCRIPTION
**Pull request by arminkollascheck**

### CODE REVIEW

File: OTGroundTruther/model/event.py [hunk 1]
```diff
@@ -216,7 +218,22 @@ def _exceeds_maximum_events(self, events: list[dict]) -> bool:
...
+        if Path(video_name).suffix == "":
+            return video_name + DEFAULT_VIDEO_FILE_SUFFIX
+        elif self._is_detection_file(video_name=video_name):
+            return Path(video_name).stem + DEFAULT_VIDEO_FILE_SUFFIX
+        else:
+            return video_name
+
+    def _is_detection_file(self, video_name: str) -> bool:
+        if (
+            video_name[-1 * len(DEFAULT_DETECTION_FILE_SUFFIX) :]
```
Comment (martinbaerwolff) [inline OTGroundTruther/model/event.py:231]: Looks complicated, can´t you get the current suffix with `Path(video_name).suffix` and compare this to `DEFAULT_DETECTION_FILE_SUFFIX`?

**Review by martinbaerwolff — CHANGES_REQUESTED** [review]
Just one comment.

+ it would be better to fix those bugs in individual branches and PR´s. The smaller the PR, the better :)

**Comment by arminkollascheck** [comment]
Sure, but I was not sure if someone will look into it anyway. I wanted a version for the students and was not sure how long the review will take.

File: OTGroundTruther/model/event.py (hunk 1 above)
Comment (arminkollascheck) [reply OTGroundTruther/model/event.py]: Dont know why i made it so complicated, lol

**Review by martinbaerwolff — APPROVED** [review]

**Pull request merged**